src/app/posts/components/post-list/post-list.component.html

```diff
@@ -0,0 +1,5 @@
+<app-post
+    class="post"
+    *ngFor = "let i of posts"
```

Comment:
(optional) use better naming: let post of posts

Reply:
Fixed in new PR Feature/hw3